src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -573,28 +573,28 @@ impl<'tcx, Tag: Copy, Extra> Allocation<Tag, Extra> {
     }
 }
 
-/// Run-length encoding of the undef mask.
+/// Run-length encoding of the uninit mask.
 /// Used to copy parts of a mask multiple times to another allocation.
-pub struct AllocationDefinedness {
+pub struct InitMaskCompressed {
     /// The definedness of the first range.
     initial: bool,
     /// The lengths of ranges that are run-length encoded.
     /// The definedness of the ranges alternate starting with `initial`.
     ranges: smallvec::SmallVec<[u64; 1]>,
 }
 
-impl AllocationDefinedness {
-    pub fn all_bytes_undef(&self) -> bool {
+impl InitMaskCompressed {
+    pub fn no_bytes_init(&self) -> bool {
         // The `ranges` are run-length encoded and of alternating definedness.
-        // So if `ranges.len() > 1` then the second block is a range of defined.
+        // So if `ranges.len() > 1` then the second block is an initialized range.
         !self.initial && self.ranges.len() == 1
     }
 }
 
 /// Transferring the definedness mask to other allocations.
```

Comment:
```suggestion
/// Transferring the initialization mask to other allocations.
```
I am stopping here; please also replace the remaining uses of "definedness" if there are any.